packages/backend-modules/calendar/graphql/resolvers/_mutations/bookCalendarSlot.js

```diff
@@ -46,6 +50,11 @@ module.exports = async (_, args, context) => {
     throw new Error(t('api/calendar/slot/error/userBookedAlready'))
   }
 
+  const isOnAllowedWeekday = calendar.limitWeekdays.includes(date.day())
```

Comment:
limitWeekdays is hardcoded to be Mo-Sa, I guess? How would we handle it if we wanted to have speakers on a Sunday exceptionally? (is that an edge case we want to think of or not?)